TableController/ITableController.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Famicom.TableController
+{
+    public interface ITableController
+    {
+        float GetTableHeight(ITable Table);
+        void SetTableHeight(ITable Table);
+        float GetTableSpeed();
+        string GetTableStatus();
+        List<ITableError> GetTableError();
```

Comment:
It might make more sense to simply expose the list publicly.
```suggestion
        public List<ITableError> {get; private set; }
```